doc/source/conf.py

```diff
@@ -73,13 +73,18 @@
               'sphinx.ext.ifconfig',
               'sphinx.ext.linkcode',
               'nbsphinx',
-              'sphinxcontrib.spelling'
               ]
 
 exclude_patterns = ['**.ipynb_checkpoints']
 
-spelling_word_list_filename = ['spelling_wordlist.txt', 'names_wordlist.txt']
-spelling_ignore_pypi_package_names = True
+# Tries to import sphixcontrib.spelling - if not available pass
+try:
+    import sphinxcontrib.spelling
+    extensions.append('sphinxcontrib.spelling')
+    spelling_word_list_filename = ['spelling_wordlist.txt', 'names_wordlist.txt']
+    spelling_ignore_pypi_package_names = True
+except ModuleNotFoundError:
```

Comment:
Can you also catch ImportError? 
Because if you have installed `sphinxcontrib.spelling`, but not pyenchant, you get this (which will be a common case I think, since this is what you get if you simply pip/conda install it)